Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs

```diff
@@ -43,6 +43,14 @@ public class ProfileSettings : ISoomlaSettings
 		static ProfileSettings()
 		{
 			SoomlaEditorScript.addSettings(instance);
+
+			List<string> additionalDependFiles = new List<string>(); //Add files that not tracked in file_list
+			additionalDependFiles.Add("Assets/Facebook");
```

Comment:
@Blinkq, are you sure that we should remove "Facebook" files? Maybe user want to remove SOOMLA but leave Facebook libs?